### ISSUE TITLE
Convert readthedocs link for their .org -> .io migration for hosted projects

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8374]
|[<img alt="adamchainz's avatar" src="https://avatars.githubusercontent.com/u/857609?s=50" width="50" height="50">](https://github.com/adamchainz)| @adamchainz reported|
|-|-|
|Trac ID|trac#8374|
|Type|enhancement|
|Created|2016-06-01 22:50:39Z|
|Branch|https://github.com/twisted/twisted/tree/8374-adamchainz-readthedocs.io|

As per their blog post of the 27th April ‘Securing subdomains’:

Starting today, Read the Docs will start hosting projects from subdomains on the domain readthedocs.io, instead of on readthedocs.org. This change addresses some security concerns around site cookies while hosting user generated data on the same domain as our dashboard.

<details><summary>Searchable metadata</summary>

```
trac-id__8374 8374
type__enhancement enhancement
reporter__adamchainz adamchainz
priority__normal normal
milestone__None None
branch__8374_adamchainz_readthedocs_io 8374-adamchainz-readthedocs.io
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1464821439086354 1464821439086354
changetime__1465162107434581 1465162107434581
version__None None
owner__adamchainz adamchainz

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adamchainz's avatar" src="https://avatars.githubusercontent.com/u/857609?s=50" width="50" height="50">](https://github.com/adamchainz)<a name="note_1"></a>|@adamchainz commented|
|-|-|

https://github.com/twisted/twisted/pull/83

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @adamchainz|
|-|-|

Looks good. Thanks!

I have sent the patch to our buildbot builders .

Waiting for build results and will merge.

Thanks again for your contribution!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset 609e96c17bedc25f28ef5743ff838f7719a79c4c
```
#!CommitTicketReference repository="" revision="609e96c17bedc25f28ef5743ff838f7719a79c4c"
Merge 8374-adamchainz-readthedocs.io: Use exec function in twisted.conch

Author: adamchainz
Reviewer: adiroiban
Fixes: #8374
```
